src/schema/v2/article/index.ts

```diff
@@ -53,26 +56,91 @@ export const ArticleType = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLList(AuthorType),
       resolve: ({ contributing_authors }) => contributing_authors,
     },
+    description: {
+      type: GraphQLString,
+    },
+    hero: {
+      type: ArticleHero,
+      resolve: ({ hero_section }) => hero_section,
+    },
     href: {
       type: GraphQLString,
       resolve: ({ slug }) => `/article/${slug}`,
     },
+    layout: {
+      type: new GraphQLNonNull(
+        new GraphQLEnumType({
+          name: "ArticleLayout",
+          values: {
+            STANDARD: { value: "standard" },
+            FEATURE: { value: "feature" },
+          },
+        })
+      ),
```

Comment:
Ah, thats how you do this. I didn't quite understand what was happening over in positrons schema in this regard